src/tests.rs

```diff
@@ -942,20 +942,35 @@ fn test_relay_signatures_can_change_reward_addresses() {
 
 		// Threshold is set to 50%, so we need at least 3 votes to pass
 		// Let's make sure that we dont pass with 2
-		let mut payload = 2u64.encode();
+		let mut payload = WRAPPED_BYTES.to_vec();
+		payload.append(&mut 2u64.encode());
 		payload.append(&mut 1u64.encode());
+		payload.append(&mut WRAPPED_BYTES.to_vec());
+
 		let mut insufficient_proofs: Vec<([u8; 32], MultiSignature)> = vec![];
 		for i in 0..2 {
 			insufficient_proofs.push((pairs[i].public().into(), pairs[i].sign(&payload).into()));
 		}
 
+		// Ensure non-root cannot change
 		assert_noop!(
 			Crowdloan::change_association_with_relay_keys(
 				Origin::signed(1),
 				2,
 				1,
 				insufficient_proofs.clone()
 			),
+			DispatchError::BadOrigin
+		);
+
+		// Not sufficient proofs presented
+		assert_noop!(
+			Crowdloan::change_association_with_relay_keys(
+				Origin::root(),
```

Comment:
DQ: I probably should have noticed this earlier. Is this a root extrinsic? So if I want to change my reward address, I not only need enough proofs, but I still need to go through democracy?

Reply:
No it was not before. But @crystalin suggested we make it root, because being compatible with tools like polkadot-js extension might be good for usability but bad for security (malicious sites could as the extension to sign a reward address change)

Reply:
I think the best we should do is to make it open for a short period of time to allow people to use it and then force it root to only use it for specific cases. Or put it under a given council (Support council ?)

Reply:
Council sounds good to me. I will then add a trait, and for now we will make it open in the runtime

Reply:
Ok now we have _RewardAddressChangeOrigin_ associated type, which dictates which are the origins allowed to call to such extrinsic. For now we will make sure in our runtime this is an EnsureSigned, but in the future we can make it a specific council